### DISCUSSION TITLE
Split screen multiplatform

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [axmolengine/axmol#1462] [General]
Hi,

Is possible create a screen divider/split like this in a way that work for all platform, not only opengl?
https://werplay.medium.com/voronoi-split-screen-a-quick-cocos2d-x-tutorial-12be81fbc98d

![image](https://github.com/axmolengine/axmol/assets/395096/0af97716-fcc0-4f36-af24-c3f1df650bc5)

Thanks.

**Comment by rh101** [comment]
> Is possible create a screen divider/split like this in a way that work for all platform, not only opengl?

That article you linked has nothing to do with OpenGL, but even if it did, it's trivial to convert the implementation to work with Axmol.

If you need to know how, then just look at changes made from Cocos2d-x v3 to Cocos2d-x v4.  Axmol is based on Cocos2d-x v4.  Any code that was OpenGL-specific in Cocos2d-x v3 was updated to work with Cocos2d-x v4, so that would be a good place to start in working out what you need to do to port OpenGL-specific code over to Axmol.
